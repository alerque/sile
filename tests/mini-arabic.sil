\begin[papersize=a6]{document}
\nofolios
\script[src=packages/bidi]
\thisframeRTL
\font[family=Amiri,language=urd,script=Arab,size=14pt]
1955 میں ان کا انتقال ہوا۔

1955 میں ان کا انتقال ہوا۔1947 میں وہ۔

1955 میں ان کا انتقال ہوا۔ 1947 میں وہ۔

میں 1955 میں آیا تھا

میں1955 میں آیا تھا
\end{document}
