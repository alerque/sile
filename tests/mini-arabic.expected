Set paper size 	297.6377985	419.5275636
Begin page
Mx 	232.9512
My 	32.0233
Set font 	Amiri;14;400;;normal;;LTR
T	20 28 24 24	(1955)
Mx 	211.9017
Set font 	Amiri;14;400;;normal;;RTL
T	3978 3974 2335	(میں)
Mx 	195.8903
T	417 386	(ان)
Mx 	180.0840
T	4764 4686	(کا)
Mx 	148.9310
T	415 2019 2189 3067 3756 386	(انتقال)
Mx 	129.7682
T	386 2440 2256	(ہوا)
Mx 	122.8844
T	559	(۔)
Mx 	232.9512
My 	48.8233
Set font 	Amiri;14;400;;normal;;LTR
T	20 28 24 24	(1955)
Mx 	211.9168
Set font 	Amiri;14;400;;normal;;RTL
T	3978 3974 2335	(میں)
Mx 	195.9204
T	417 386	(ان)
Mx 	180.1291
T	4764 4686	(کا)
Mx 	148.9912
T	415 2019 2189 3067 3756 386	(انتقال)
Mx 	129.8434
T	386 2440 2256	(ہوا)
Mx 	122.9596
T	559	(۔)
Mx 	93.1550
Set font 	Amiri;14;400;;normal;;LTR
T	20 28 23 26	(1947)
Mx 	72.1205
Set font 	Amiri;14;400;;normal;;RTL
T	3978 3974 2335	(میں)
Mx 	56.4113
T	540 419	(وہ)
Mx 	49.5275
T	559	(۔)
Mx 	232.9512
My 	65.6233
Set font 	Amiri;14;400;;normal;;LTR
T	20 28 24 24	(1955)
Mx 	211.9178
Set font 	Amiri;14;400;;normal;;RTL
T	3978 3974 2335	(میں)
Mx 	195.9224
T	417 386	(ان)
Mx 	180.1322
T	4764 4686	(کا)
Mx 	148.9952
T	415 2019 2189 3067 3756 386	(انتقال)
Mx 	129.8485
T	386 2440 2256	(ہوا)
Mx 	122.9647
T	559	(۔)
Mx 	88.2320
Set font 	Amiri;14;400;;normal;;LTR
T	20 28 23 26	(1947)
Mx 	67.1986
Set font 	Amiri;14;400;;normal;;RTL
T	3978 3974 2335	(میں)
Mx 	51.4903
T	540 419	(وہ)
Mx 	44.6066
T	559	(۔)
Mx 	246.6504
My 	83.3576
T	3978 3974 2335	(میں)
Mx 	211.8940
Set font 	Amiri;14;400;;normal;;LTR
T	20 28 24 24	(1955)
Mx 	190.8367
Set font 	Amiri;14;400;;normal;;RTL
T	3978 3974 2335	(میں)
Mx 	176.9572
T	2019 2107 381	(آیا)
Mx 	160.6578
T	2019 6169 2094	(تھا)
Mx 	246.6504
My 	100.8186
T	3978 3974 2335	(میں)
Mx 	216.8458
Set font 	Amiri;14;400;;normal;;LTR
T	20 28 24 24	(1955)
Mx 	195.7875
Set font 	Amiri;14;400;;normal;;RTL
T	3978 3974 2335	(میں)
Mx 	181.9070
T	2019 2107 381	(آیا)
Mx 	165.6065
T	2019 6169 2094	(تھا)
End page
Finish
